manager/manager.go

```diff
@@ -658,7 +656,9 @@ func (m *manager) GetMachineInfo() (*info.MachineInfo, error) {
 }
 
 func (m *manager) GetVersionInfo() (*info.VersionInfo, error) {
-	return &m.versionInfo, nil
+	// TODO: Consider caching this and periodically updating.  The VersionInfo may change if
+	// the docker daemon is started after the cAdvisor client is created
```

Comment:
nit: Add to TODO - Might also be helpful to cache, so we can return last known docker version if docker is down at the time of query.